smslibrary/src/main/java/com/eis/smslibrary/SMSMessage.java

```diff
@@ -87,7 +87,7 @@ public SMSPeer getPeer() {
      * so that one can see all the possible values without having to
      * look at the official Android documentation
      * <p>
-     * These are used in {@link SMSMessageSentListener}
+     * These are used in {@link SMSSentBroadcastReceiver}
```

Comment:
These states are used in the listener, the user doesn't have to care about the BroadcastReceiver, he just has to know that these states are passed as parameter inside the listener and he can use them there

Reply:
Alright, since it wasn't something I worked on and since the link had an error (due to a class rename), I just put the class where the states were used. I'll update it now 👍